streamlit_demo/app.py

```diff
@@ -0,0 +1,51 @@
+import os
+import streamlit as st
+import soundfile as sf
+
+import numpy as np
+
+import librosa
+import pysinsy
+from nnmnkwii.io import hts
+from nnsvs.svs import SPSVS
+
+st.title("NNSVS Demo")
+st.markdown("Upload your .xml music file with text as input to make it singed.")
+
+models = {
+  'kiritan': '20220321_kiritan_timelag_mdn_duration_mdn_acoustic_resf0conv',
+  'yoko' : '20220322_yoko_timelag_mdn_duration_mdn_acoustic_resf0conv'
+}
+
+voice_option = st.selectbox(
+     'Select the voice',
+     models.keys()
+)
+uploaded_file = st.file_uploader("Choose a .xml music file", type='xml')
+
+if uploaded_file:
```

Comment:
What about having a `synthesis` button to run SVS engine explicitly? Like: https://github.com/r9y9/ttslearn/blob/4939934554daadd0bd85c995e70d88f58206056e/demo_server.py#L102

Reply:
I put it, but personally I think that is an useful action required to user without real motivation for actual version of the demo. maybe if you have to many params you want to change more than one before play it is very useful.